tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTestBase.java

```diff
@@ -86,6 +89,7 @@ public void before() {
         lraMetricService.clear();
         this.lraClient = lraTestService.getLRAClient();
         this.tckSuiteTarget = lraTestService.getTCKSuiteTarget();
+        this.lraManagementService = LRATestService.loadService(LRAManagementService.class);
```

Comment:
I'm thinking if it's cleaner having service loader to be used only from one place - ie. changing the `loadService` to `private` and pass all the responsibility for loading and providing SPI services only to `LRATestService`?